source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRColumnRenderer.mm

```diff
@@ -6,48 +6,46 @@
 //
 
 #import "ACRColumnRenderer.h"
+#import "ACRColumnView.h"
 #import "ACRRegistration.h"
 #import "Column.h"
 #import "SharedAdaptiveCard.h"
 
 @implementation ACRColumnRenderer
 
-+ (ACRColumnRenderer* ) getInstance
```

Comment:
Which one is that, and why? having no spaces at all makes it somewhat harder to read.

---
In reply to: [128594827](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128594827) [](ancestors = 128594827,128563718,128559390)